### PR TITLE
Update docs: fix typos

### PR DESCRIPTION
**Pull request by KyryloKilin**

### CODE REVIEW

File: docs/operate/config-reference.mdx [hunk 1]
```diff
@@ -428,7 +428,7 @@ export HYP_CHAINS_${CHAIN_NAME}_SIGNER_TYPE="hexKey"
 
 **Requires:** `chains.<chain_name>.signer.type = "hexKey" | undefined`
 
-**Optional:** No (iff requirements are met)
+**Optional:** No (if requirements are met)
```
Comment (edakturk14) [inline docs/operate/config-reference.mdx:431]: "iff" means "if and only if," which is more precise in this context and with the other ones on the same file

**Review by edakturk14 — COMMENTED** [review]
thank you for contributing to the docs!

File: docs/operate/config-reference.mdx (hunk 1 above)
Comment (KyryloKilin) [reply docs/operate/config-reference.mdx]: Thank you!
Completed ✅

**Comment by edakturk14** [comment]
Thank you @KyryloKilin!!

**Pull request merged**